users/clients.go

```diff
@@ -62,4 +63,12 @@ type ClientService interface {
 
 	// Identify returns the client id from the given token.
 	Identify(ctx context.Context, tkn string) (string, error)
+
+	// IssueToken issues a new access and refresh token.
+	IssueToken(ctx context.Context, identity, secret string) (*mainflux.Token, error)
+
+	// RefreshToken refreshes expired access tokens.
+	// After an access token expires, the refresh token is used to get
+	// a new pair of access and refresh tokens.
```

Comment:
After Refresh token expired? how we will get new refresh token ? By Login Again?

Reply:
Yes. 

The process is usually login get access and refresh token use refresh token to renew the access token. If inactivity is past the refresh token period you log in again.